data/elasticsearch/elasticsearch.go

```diff
@@ -23,26 +24,67 @@ func (es *ElasticSearch) Connect(url string) error {
 	return err
 }
 
-// Index stores a document inside elastic search
-func (es *ElasticSearch) Index(esIndex string, esType string, body interface{}) error {
+// CreateIndexIfNotExists Creates an index if it doesn't already exists
```

Comment:
I had removed it in the previous PR that was merged and you added it back. I guess you can remove it in your next PR.